configuration/scripts/machines/env.compy_intel

```diff
@@ -0,0 +1,42 @@
+#!/bin/csh -f
+
+set inp = "undefined"
+if ($#argv == 1) then
+  set inp = $1
+endif
+
+if ("$inp" != "-nomodules") then
+
+source /share/apps/modules/init/csh
+
+module purge
+module load intel/19.0.3
+module load gcc/8.1.0
+module load intelmpi/2019u3
+module load netcdf/4.6.3 
+module load hdf5/1.10.5
+
+#setenv NETCDF_PATH ${NETCDF_DIR}
+setenv NETCDF_PATH /share/apps/netcdf/4.6.3/intel/19.0.3
+setenv OMP_PROC_BIND true
+setenv OMP_PLACES threads
+limit coredumpsize unlimited
+limit stacksize unlimited
+
+endif
+
+setenv ICE_MACHINE_MACHNAME compy
+setenv ICE_MACHINE_MACHINFO "Intel Xeon Skylake with 192 GB of DDR4 DRAM"
+setenv ICE_MACHINE_ENVNAME intel
+setenv ICE_MACHINE_ENVINFO "ifort 19.0.3"
```

Comment:
Propose this be
setenv ICE_MACHINE_ENVINFO "intel/19.0.3 gcc/8.1.0 intelmpi/2019u3 netcdf/4.6.3"

Reply:
I have changed this to your suggested entry.